src/components/KRadio/KRadio.vue

```diff
@@ -10,106 +10,196 @@
       type="radio"
       @click="handleClick"
     >
-    <span
+    <div
+      v-if="type === 'card' && $slots.before"
```

Comment:
suggestion: Instead of checking for `type === 'card` or `type === 'default` in so many places, can you map this to a computed variable and do something like this instead? Then you can test the computed in the `template` section.

```ts
const isDefaultType = computed((): boolean => props.type === 'default`)
```